deltachat-ios/Helper/DatabaseHelper.swift

```diff
@@ -0,0 +1,94 @@
+import Foundation
+class DatabaseHelper {
+
+    var sharedDbFile: String {
+        guard let fileContainer = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: "group.eu.merlinux.group.chat.delta.ios") else {
```

Comment:
can you please move `group.eu.merlinux.group.chat.delta.ios` to a constant and add a comment about where this string comes from and why it is needed this way. also this "double" `group.` part needs some explanations.

Reply:
I've moved the group id to a constant. The double group is unnecessarey IMO. I only reused the preconfigured one that was already available.

Reply:
I also added the explanation